x-pack/plugins/security_solution/cypress/integration/timelines/row_renderers.spec.ts

```diff
@@ -98,7 +98,7 @@ describe('Row renderers', () => {
           expect(
             elementsOverlap($googleLinkTooltip, $signatureTooltip),
             'tooltips do not overlap'
-          ).to.equal(false);
+          ).to.equal(true);
```

Comment:
Are the tool tips overlapping now with that change? Seems like you had to switch from false to true?

Reply:
I ran the cypress tests in your fork locally and this test fails with `to.equal(true)` as it should. The test was created to make sure that the two tooltips on this link don't overlap:

<img width="1683" alt="Screenshot 2021-12-08 at 10 23 00" src="https://user-images.githubusercontent.com/68591/145187170-26543d1b-7a3b-4d91-a173-3750b713617e.png">

Changing it back to `to.equal(false)` fixes the tests for me locally. Does the test break on the CI with `to.equal(false)`?

Reply:
I've been testing it locally (Cypress is awesome, and the docs are super helpful) with the same results you're getting. I'll change it back and update the three others that are failing. I'm also seeing those failures locally with a browser. I'll try them locally headless and hopefully get this finished up.